src/consts.hpp

```diff
@@ -57,7 +57,7 @@ inline constexpr madrona::CountT numPhysicsSubsteps = 0.f;
 inline constexpr float zDimensionScale = 1;
 inline constexpr float xDimensionScaleRoadSegment = 1;
 
-inline constexpr madrona::CountT kTrajectoryLength = 91; // Nocturne has 90 timesteps per episode. making it 91 as a buffer.
+// inline constexpr madrona::CountT kTrajectoryLength = 91; // Nocturne has 90 timesteps per episode. making it 91 as a buffer.
```

Comment:
Even if we support variable episode lengths, the log replay trajectories in the scenes will remain of shape 91. 

Why is this deleted? There is an important difference between the length of the episode and the length of the data in the simulator